### ISSUE TITLE
Domain mapping warning message

### ISSUE THREAD

**Issue by adelineyaw** [Automattic/wp-calypso#56376]
### Quick summary

A HC has submitted this request: pd4QKa-gK-p2 re: domain mapping messaging.

TL; DR:

"Users are confused about this message and ask us why their domain name stopped working. Maybe some of them don’t know what does the term “domain mapping” mean.

It would be helpful if we replace this message with another clearer one, so that those users will be self-guided, and don’t need to ask for our help."

### Steps to reproduce

If you visit a WordPress.com site that has a broken or deactivated domain connection, you'll be able to reproduce this error as seen here: 

![warning_domain_mapping](https://user-images.githubusercontent.com/7424885/133908196-8ae3756f-9310-451c-b723-d17d78cbe3e0.png)

### What you expected to happen

The HC suggested the following to replace the current message:

_`Your website doesn’t have a paid plan or a domain connection upgrade. Domain Connection is available only for websites that have a paid plan. You should be able to go to Manage Purchases to renew your plan or your domain connection upgrade.`_

This is just a generic sample, but we can customize that message to be more precise, so that it matches with each user-case properly; whether they had an expired plan or if their domain mapping upgrade has expired.

Example ticket: 4299692-zen

It's also possible that the domain connection has been removed or deactivated so we'll need to somehow incorporate all the possibilities of why users are unable to see their WordPress.com site at a particular domain.

### What actually happened

N/A

### Context

HC request: pd4QKa-gK-p2

**Comment by delputnam** [comment]
Just noting that I have created a task for Nomado to handle this issue.  We will update the issue once the task is completed.

**Comment by leonardost** [comment]
Hi! We've updated the message to hopefully be more clear and helpful to users, the diff is in D68180-code. It looks like this now:

<img width="823" alt="Screen Shot 2021-10-12 at 16 50 27" src="https://user-images.githubusercontent.com/5324818/137021740-f8297522-36ea-4dbf-8b06-c94749925efb.png">

It now contains a link to the [domain connection](https://wordpress.com/support/domains/connect-existing-domain/) support page and hints that the error might be appearing because the user's plan or domain have expired. I'll close this issue now.
